public/js/app.js

```diff
@@ -20,6 +20,87 @@ jQuery( document ).ready(function( $ ){
             });
         });
         $('.comment-btn').on('click', makeComment);
+
+        $('#followsLink, #followersLink').click(function(e) {
+            e.preventDefault();
+            var url = $(this).attr('data-url');
+            var title = /follows/.test(url) ? 'Following' : 'Followers';
+
+            $.getJSON(url, function(data) {
+                if (data.length == 0)
+                    return;
+
+                $('#ajaxModal').on('show.bs.modal', function(event) {
+                    var modal = $(this);
+                    modal.find('.modal-title').text(title);
+
+                    var html = "";
+
+                    data.forEach(function(user) {
+                        follow = user.checkFollow ? 'Following' : 'Follow';
+                        display = user.me ? "style='display:none'" : '';
+
+                        html += "<div class='row follows'>"
+                             + "<div class='col-md-9'>"
+                             + "<img align='left' class='img-circle img-responsive' src='" + user.avatar + "' alt='Profile image' border-radius='100%''>"
+                             + "<span><a href='/" + user.username +"'>" + user.username + "</a></span>"
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Bad line breaking before '+'.